js_modules/dagster-ui/packages/ui-core/src/hooks/useSetStateUpdateCallback.tsx

```diff
@@ -33,3 +33,31 @@ export function useSetStateUpdateCallback<T>(
 
   return update;
 }
+
+export function usePartialSetStateUpdateCallback<T>(
```

Comment:
Add a comment with a bit more about when to use this vs `useSetStateUpdateCalback`?